Scripts/Runtime/Debug/UnityTraceListener.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using UnityEngine;
+
+namespace Anvil.Unity.Debugging
```

Comment:
I'm open to suggestions on the folder/namespace naming. It just can't be `Anvil.Unity.Debug` for naming conflict reasons.

Reply:
I think this works, the `csharp` library has a `Anvil.CSharp.Debugging` namespace for `AssertUtil` so i think we're being consistent.